backend/modules/website/services/website.service.ts

```diff
@@ -0,0 +1,54 @@
+import WebsiteDao from "../daos/website.dao";
+import { CreateWebsiteDto, UpdateWebsiteDto } from "../dto/create.website.dto";
+import { GetWebsitesDto, GetWebsiteDto, DeleteWebsiteDto } from "../dto/get.websites.dto";
+
+
+class WebsiteService {
+
+  async getWebsiteById(getWebsiteByIdPayload: GetWebsiteDto) {
+    try {
```

Comment:
Service is supposed to receive params suffixed with payload and prepare data that goes into dao and hits the query.

DAO should not deal with data modification to make it query friendly, service should do that job.